{"name": "Создание задачи с невалидной датой", "status": "passed", "steps": [{"name": "Отправляем POST запрос с невалидной датой: [123]", "status": "passed", "start": 1746535612324, "stop": 1746535612838}, {"name": "Проверяем код ответа и сообщение об ошибке", "status": "passed", "start": 1746535612838, "stop": 1746535612838}], "parameters": [{"name": "wrong_jsons", "value": "{'name': 'Test', 'due_date': [123]}"}], "start": 1746535612323, "stop": 1746535612838, "uuid": "e7d4fdbb-d50f-46d3-86bd-235e88788f71", "historyId": "8a9cd3f54c9cbbdf0a834914a91bcfee", "testCaseId": "cdf5097d6132b1714cb61dcaa05cd861", "fullName": "tests.api.test_tasks.TestTasks#test_create_task_invalid_date", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "Egors-MacBook-Air.local"}, {"name": "thread", "value": "21580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}